app/models/claims/claim_window.rb

```diff
@@ -0,0 +1,75 @@
+# == Schema Information
+#
+# Table name: claim_windows
+#
+#  id               :uuid             not null, primary key
+#  discarded_at     :date
+#  ends_on          :date
+#  starts_on        :date
+#  created_at       :datetime         not null
+#  updated_at       :datetime         not null
+#  academic_year_id :uuid             not null
+#
+# Indexes
+#
+#  index_claim_windows_on_academic_year_id  (academic_year_id)
+#  index_claim_windows_on_discarded_at      (discarded_at)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (academic_year_id => academic_years.id)
+#
+class Claims::ClaimWindow < ApplicationRecord
+  include Discard::Model
+
+  default_scope -> { kept }
+
+  belongs_to :academic_year, optional: true
+
+  validates :starts_on, presence: true
+  validates :ends_on, presence: true
```

Comment:
Should included comparison validation:
validates :ends_on, comparison: { less_than: :starts_on }

Reply:
![screencapture-track-and-pay-854-test-teacherservices-cloud-support-claim-windows-9af23a63-bb79-45c8-bfeb-06aec385f4d4-check-2024-07-18-15_34_24](https://github.com/user-attachments/assets/a707dfdb-0f12-452f-8d9b-fb67aefccc38)